cns/grpc/proto/server.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package cns;
+
+option go_package = "grpc/protogen";
+
+// The Container Network Service (CNS) exposes a set of operations that allow the Distributed Network Controller (DNC) to manage
```

Comment:
Correcting the naming :)

```suggestion
// The Container Network Service (CNS) exposes a set of operations that allow the Delegated Network Controller (DNC) to manage
```

Reply:
Hey Tim, just implemented this change thank you!